src/Build/BackEnd/Components/Logging/EventSourceSink.cs

```diff
@@ -224,57 +224,59 @@ public void Consume(BuildEventArgs buildEvent, int sinkId)
         /// </summary>
         public void Consume(BuildEventArgs buildEvent)
         {
+
             switch (buildEvent)
             {
                 case BuildMessageEventArgs buildMessageEvent:
-                    RaiseMessageEvent(null, buildMessageEvent);
+                    RaiseEvent(null, buildMessageEvent, (o, args) => MessageRaised?.Invoke(o, args), (o, args) => AnyEventRaised?.Invoke(o, args));
```

Comment:
the follow up is  AnyEventRaised here, but in RaiseMessageEvent is RaiseAnyEvent - AnyEventRaised with try catch block of its own. don't we still need it?